### ISSUE TITLE
Make "restart TS server" script available in Svelte files

### ISSUE THREAD

**Issue by PierreTurnbull** [microsoft/vscode#210102]
According to [this SO post](https://stackoverflow.com/a/64456017/8993709), the "restart TS server" script is only available when a .js, .ts or .tsx file is focused. Could we have it available when a .svelte file is focused please? A Svelte file usually contains some JS or TS:
```svelte
<script lang="ts">
    let str: string = "Hello world!"
</script>

<p>{str}</p>
```

**Comment by mjbvz** [comment]
Are they using the TS Server through a plugin?

**Comment by PierreTurnbull** [comment]
No I don't have any TS extension installed.

**Comment by PierreTurnbull** [comment]
@mjbvz No I don't have any TS extension installed.

**Comment by mjbvz** [comment]
Sorry if this was unclear. To be more precise: does the svelte extension work by using our built-in TS Server?

**Comment by PierreTurnbull** [comment]
@mjbvz thank you for the clarification. I searched a bit and I found a "Svelte: restart language server" which seems to do exactly what I was asking for in the first place. 

When using this command, I noticed the TS errors and warnings in the code disappeared and reappeared, which means that the TS server used in Svelte files is restarted.
I suppose that this command restarts the TS server of VSCode under the hood. Or perhaps it's a completely distinct language server that incorporates its own TS. I don't know.

Anyway, problem solved, the feature I was requesting already exists (just not in the way I was expecting!)
